src/MLModal/MLModal.js

```diff
@@ -0,0 +1,5 @@
+import { DraggableModal } from 'ant-design-draggable-modal'
+
+const MLModal = DraggableModal
```

Comment:
Is there any reason not to follow the wrapper style of the other components? We should prefer that, because it lets us set the `ml-` className, and keeps the library consistent in structure.

Here's the code I recommend using unless it breaks something I don't know about:
```
import React from 'react'
import { DraggableModal } from 'ant-design-draggable-modal'
import classNames from 'classnames'

const MLModal = (props) => {
  return (
    <DraggableModal
      {...props}
      className={classNames('ml-modal', props.className)}
    >
      {props.children}
    </DraggableModal>
  )
}

export default MLModal
```
```

Reply:
I like to opt for less code when possible, and it makes it extra clear that there's no difference from the Ant component. Consistency is also important though, and if we're setting classnames then that's reason enough to use a wrapper. Fixed.